pkg/k8s/client.go

```diff
@@ -2,19 +2,20 @@ package k8s
 
 import (
 	"fmt"
+	"os"
 
 	accessclient "github.com/servicemeshinterface/smi-sdk-go/pkg/gen/client/access/clientset/versioned"
 	specsclient "github.com/servicemeshinterface/smi-sdk-go/pkg/gen/client/specs/clientset/versioned"
 	splitclient "github.com/servicemeshinterface/smi-sdk-go/pkg/gen/client/split/clientset/versioned"
 	"github.com/sirupsen/logrus"
-	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 // Client is an interface for the various resource controllers.
 type Client interface {
-	GetKubernetesClient() kubeclient.Interface
+	GetKubernetesClient() kubernetes.Interface
```

Comment:
Maybe we could remove the `Get` prefix in client getters name?

As described [here](https://golang.org/doc/effective_go.html#Getters), the getter method name should be `KubernetesClient()`.

> There's nothing wrong with providing getters and setters yourself, and it's often appropriate to do so, but it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner.